code/addons/themes/docs/getting-started/material-ui.md

```diff
@@ -0,0 +1,112 @@
+# 🏁 Getting started with `@mui/material`
+
+## 📦 Install addon
+
+**NOTE:** As of Storybook 7.2, `@storybook/addon-themes` ships in `@storybook/addon-essentials`. Only follow this step if you are using Storybook >= 7.0 && < 7.2
+
+To get started, **install the package** as a dev dependency
+
+yarn:
+
+```zsh
+yarn add -D @storybook/addon-themes
+```
+
+npm:
+
+```zsh
+npm install -D @storybook/addon-themes
+```
+
+pnpm:
+
+```zsh
+pnpm add -D @storybook/addon-themes
+```
+
+## 🧩 Register Addon
+
+Now, **include the addon** in your `.storybook/main.js` file
+
+```diff
+module.exports = {
+  stories: [
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)",
+  ],
+  addons: [
+    "@storybook/addon-essentials",
++   "@storybook/addon-themes",
```

Comment:
This step is also optional if the user has SB 7.2 and above

Reply:
Good catch <3